runtime/src/sorted_storages.rs

```diff
@@ -7,7 +7,7 @@ use std::ops::Range;
 pub struct SortedStorages<'a> {
     range: Range<Slot>,
     storages: Vec<Option<&'a SnapshotStorage>>,
-    count: usize,
+    slot_count: usize,
```

Comment:
[Amount I care: 1/5] Wondering what prompted the rename to `slot_count`? Since the name of the struct is about storages, the count intuitively felt like a count of storages to me.

I know the number of slots and storages are the same, so no question in that regard.

Reply:
this is only used for returning a metric. The metric used to count the # of append vecs across all slots. Assuming we have only 1, we now only count the # of slots with non-empty values, which will return an equivalent # practically, but means something different.